stream-chat-android-core/build.gradle

```diff
@@ -31,6 +31,8 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
 }
 
 dependencies {
+    api project(":libraries:logging:stream-logger")
```

Comment:
Shouldn't we include it only as `debugApi`?

Reply:
`core` contains a dependency on a `stream-logger` which is just an entry point and does not contain any logging behind.

`StreamLog` is disabled by default